app/models/import.rb

```diff
@@ -58,17 +58,17 @@ def generate(file, proof_report)
         end
 
       rescue Exception => e
-        Rails.logger.info "#"*50
-        Rails.logger.info e.inspect
         valid = false
+        puts e.inspect
```

Comment:
Making these `Rails.logger.info` instead of `puts` would make it so they show up on -d.  Could you change these?

Reply:
Updated